Terasology/LightAndShadowResources
# feat: new background asset for ammo slot

## Changes 

Placed a new background for Ammo slot:

- New Asset
![ammo](https://user-images.githubusercontent.com/48293545/91633951-3abdd480-e9f5-11ea-8c89-f54353a6d8e5.png)

- Other tries
![image](https://user-images.githubusercontent.com/48293545/91633992-78baf880-e9f5-11ea-9a2a-1608af70cfda.png)


## How to test

1. Check [link]
2. Start a LAS game
3. Acquire a bow and some arrows


## Result:
![image](https://user-images.githubusercontent.com/48293545/91633896-b1a69d80-e9f4-11ea-905a-052fa3e15f62.png)
![image](https://user-images.githubusercontent.com/48293545/91633941-27ab0480-e9f5-11ea-9a94-335b36057a03.png)
